apps/total-typescript/src/components/team-invites.tsx

```diff
@@ -0,0 +1,100 @@
+import * as React from 'react'
+import {format, parseISO} from 'date-fns'
+import get from 'lodash/get'
+import isEmpty from 'lodash/isEmpty'
+import useClipboard from 'react-use-clipboard'
+import Tippy from '@tippyjs/react'
```

Comment:
if we remove this component, then we can remove the Tippy dependency as well

Reply:
I'm leaving that for the team work that needs to happen